06_gpu_and_ml/llm-serving/chat_with_pdf_vision.py

```diff
@@ -36,7 +36,9 @@
     .apt_install("git")
     .pip_install(
         [
-            "git+https://github.com/illuin-tech/colpali.git@782edcd50108d1842d154730ad3ce72476a2d17d",  # we pin the commit id
+            "colpali-engine==0.3.5",  # Use the stable PyPI version instead of git
+            "transformers>=4.45.0",   # Ensure compatible transformers version
+            "torch>=2.0.0",          # Ensure compatible torch version
```

Comment:
can we remove the comments:
```suggestion
            "colpali-engine==0.3.5",
            "transformers>=4.45.0",
            "torch>=2.0.0",
```